packages/mdc-theme/_variables.scss

```diff
@@ -33,7 +33,7 @@ $mdc-theme-on-primary: if(mdc-theme-contrast-tone($mdc-theme-primary) == "dark",
 
 // The $mdc-theme-accent variable is DEPRECATED - it exists purely for backward compatibility.
 // The $mdc-theme-secondary* variables should be used for all new projects.
-$mdc-theme-accent: #018786 !default; // baseline teal, 600 tone
+$mdc-theme-accent: #03dac6 !default; // baseline teal, 600 tone
```

Comment:
This is a bit nit picky, but is the comment "600 tone" still accurate?

Reply:
Oh, yes that's good question.

I used [color tool](https://material.io/design/color/the-color-system.html#tools-for-picking-colors) to find tone number.

Updated to 200.